content_cafe.py

```diff
@@ -33,7 +35,9 @@
 
 from core.mirror import MirrorUploader
 
-class ContentCafeCoverageProvider(BibliographicCoverageProvider):
+from coverage_utils import MetadataWranglerBibliographicCoverageProvider
+
+class ContentCafeCoverageProvider(MetadataWranglerBibliographicCoverageProvider):
```

Comment:
This is just a BibliographicCoverageProvider that overrides `work` to create an INTERNAL_PROCESSING dummy LicensePool if necessary rather than creating a LicensePool that comes from the DataSource of the CoverageProvider (which in this case would be Content Cafe).